test/test_conversion_driver_esm1p5.py

```diff
@@ -130,29 +131,28 @@ def test_convert_fields_file_list_success(mock_process,
 
 
 def test_convert_fields_file_list_fail_excepted(mock_process_with_exception):
-    # Hopefully this test will be unnecessary with um2nc standalone.
-    # Test that the "Variable can not be processed" error arising from time
-    # series inputs is excepted.
-    allowed_error_message = esm1p5_convert.ALLOWED_UM2NC_EXCEPTION_MESSAGES[
-        "TIMESERIES_ERROR"
-    ]
-    mock_process_with_exception(allowed_error_message)
+    # Test that UnsupportedTimeSeriesErrors are excepted.
+
+    # TODO: This test will not catch changes to the exceptions in um2nc. E.g.
+    # if decided to instead raise a RuntimeError in um2nc when timeseries
+    # are encountered, the conversion of ESM1.5 outputs would undesirably
+    # crash, but this test would say everything is fine, since we're
+    # prescribing the error that is raised.
```

Comment:
You could define the error type raised in this instance as a global variable in `conversion_driver_esm1p5.py`, and then import it for use in the test as the expected error type?